jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/bars/SystemBarsTests.java

```diff
@@ -63,4 +65,41 @@ public void iconBarTests() {
         redColoredSystemBar.batteryIcon().is().displayed();
         redColoredSystemBar.timeSpan().has().text("12:30");
     }
+
+    @Test
+    public void themeBarTest() {
+        lightLightsOutSystemBar.show();
+        lightLightsOutSystemBar.has().lightTheme();
+        darkLightsOutSystemBar.has().darkTheme();
+    }
+
+    @Test
+    public void heightBarTest() {
+        highSystemBar.show();
+        highSystemBar.has().heightGreaterThan(90);
+        highSystemBar.has().heightLessThan(110);
+        highSystemBar.has().height(100);
+    }
+
+    @Test
+    public void lightsOutBarTest() {
+        lightLightsOutSystemBar.show();
+        lightLightsOutSystemBar.is().lightsOut();
+        darkLightsOutSystemBar.is().lightsOut();
+        windowSystemBar.is().notLightsOut();
+    }
+
+    @Test
+    public void windowBarTest() {
+        windowSystemBar.show();
+        windowSystemBar.is().window();
+        absolutePositionSystemBar.show();
+        absolutePositionSystemBar.is().notWindow();
+    }
+
+    @Test
+    public void titleBarTest() {
+        absolutePositionSystemBar.show();
+        absolutePositionSystemBar.has().titleText("System bar title");
```

Comment:
absolutePositionSystemBar.title().has().text("System bar title");

+ title can be in v-toolbar__title or in v-app-bar-title__content